src/persist-txn/src/txns.rs

```diff
@@ -811,19 +812,19 @@ mod tests {
         // timestamp.
         let mut ts = 0;
         while ts < 32 {
-            info!("{} direct", ts);
             subs.push(txns.read_cache().expect_subscribe(&client, d0, ts));
             ts += 1;
+            info!("{} direct", ts);
+            txns.begin().commit_at(&mut txns, ts).await.unwrap();
```

Comment:
This happened to work before with just the data shard being written and not the txns shard. Seems like a nice property to have, but it's tricky with the new approach (for subtle and uninteresting reasons that I already forgot over thanksgiving) and it doesn't really matter for how we expect this to used in mz, so may as well make it easy on ourselves